.github/workflows/build.yml

```diff
@@ -75,9 +78,10 @@ jobs:
         run: ./${{ matrix.artifact_name }} -h
 
   zip:
```

Comment:
`zip_and_upload` may be a better name for this (although this was not modified in this PR)

Reply:
updated this and the 'push' check, thanks!